docs/add-merge-automation-to-your-repository.md

```diff
@@ -0,0 +1,38 @@
+# Adding merge automation to your repository
+
+In order to run tests and enable automated merging of pull requests for your repository on KubeVirt [prow] you need to have the following components configured:
+- [tide] - responsible for maintaining the merge pool
+- [lgtm] - sets `lgtm` label
+- approve - sets label `approved`
+- [trigger] - runs test jobs for a PR
+- [branch-protection] tide takes from it required status for the [presubmit] jobs configured for the repository
+
+In a nutshell: if tide is enabled then for any created PR that has `lgtm` and `approved` labels and all presubmit checks that are required are successful, tide will merge it. 
+
+Example configuration PR to enable tide for KubeVirt [prow]: https://github.com/kubevirt/project-infra/pull/653/
+
+Example workflow:
+- tide is configured for PRs on a repository to require `lgtm` and `approved` labels and missing `needs-rebase` label
+- Alice creates a pull request, which triggers presubmit jobs to run for the PR
+- some presubmit jobs have `optional: false` set, which makes their status required to be successful
```

Comment:
some presubmit jobs have `optional: false` set, which then requires their status to be successful